itou/utils/constants.py

```diff
@@ -27,5 +27,7 @@
 # ------------------------------------------------------------------------------
 DAYS_OF_GRACE = 30
 GRACE_PERIOD = datetime.timedelta(days=DAYS_OF_GRACE)
-DAYS_OF_INACTIVITY = 365 * 2 - DAYS_OF_GRACE
+EXPIRATION_DAYS = 365 * 2
+EXPIRATION_PERIOD = datetime.timedelta(EXPIRATION_DAYS)
```

Comment:
```suggestion
EXPIRATION_PERIOD = datetime.timedelta(days=EXPIRATION_DAYS)
```
a le mérite d'être plus clair/non-ambigu